src/main/java/seedu/budgetbuddy/Storage.java

```diff
@@ -361,7 +362,36 @@ public void saveSplitExpenses(List<SplitExpense> splitExpenses) throws IOExcepti
                 splitExpense.getDescription()));
         }
         writer.close();
-    }    
+    }
+
+    public List<Budget> loadBudgets() throws FileNotFoundException {
```

Comment:
The method throws a FileNotFoundException which is adequate :) Would it be better to throw a more general IOException considering file operations can fail due to various reasons?